django_rq/queues.py

```diff
@@ -22,6 +23,28 @@ def get_commit_mode():
     return RQ.get('AUTOCOMMIT', True)
 
 
+def get_queue_class(config):
+    """
+    Return queue class from config or from RQ settings, otherwise return DjangoRQ
+    """
+    RQ = getattr(settings, 'RQ', {})
+    queue_class = DjangoRQ
+    if 'QUEUE_CLASS' in config:
+        queue_class = config.get('QUEUE_CLASS')
+    elif 'QUEUE_CLASS' in RQ:
+        queue_class = RQ.get('QUEUE_CLASS')
+
+    try:
```

Comment:
Sorry for the late reply. Django >= 1.5 bundles `six` so we can remove the `try/except` statement here since the we support Django >= 1.5.